### ISSUE TITLE
Using {envpython} in a tox configuration file does not work on MINGW

### ISSUE THREAD

**Issue by eine** [msys2/MINGW-packages#7261]
Coming from #7014.

https://github.com/msys2/MINGW-packages/issues/7014#issuecomment-724049058
> * [eine/vunit/runs/1374519423?check_suite_focus=true#step:6:11](https://github.com/eine/vunit/runs/1374519423?check_suite_focus=true#step:6:11)
> * [eine/vunit@`5ffcf09`/.github/workflows/push.yml#L99-L136](https://github.com/eine/vunit/blob/5ffcf09d8b55af7cca863922074c8ee2206ce047/.github/workflows/push.yml#L99-L136)

Steps for reproducing locally: https://github.com/msys2/MINGW-packages/issues/7014#issuecomment-724117458

https://github.com/msys2/MINGW-packages/issues/7014#issuecomment-724137686
> This issue looks like `{envpython}` doesn't work with MINGW, it is looking for python in Scripts instead of bin.
> If you run tox on a project that doesn't use {envpython} in the config, it should work fine. We should try to work with the tox project to fix this upstream.

/cc @danyeaw @gaborbernat

**Comment by revelator** [comment]
Yep i noticed this as well, seems some python ports still use the original windows build locations where scripts is indeed located in the root of the directory eg. C:\python\scripts

**Comment by eine** [comment]
Surprisingly, this is now working on CI if tox is installed through pip, but not through pacman. See https://github.com/eine/vunit/actions/runs/865359040 (https://github.com/eine/vunit/blob/ci/tox/.github/workflows/tox.yml). However, I cannot reproduce it locally. This is the error I get locally, on a fresh MSYS2 installation:

```sh
# tox -e py39-lint -- --color=yes
.package recreate: T:/vunit/vunit/.tox/.package
.package installdeps: setuptools >= 35.0.2, setuptools_scm >= 2.0.0, <3
ERROR: invocation failed (exit code 1), logfile: T:/vunit/vunit/.tox/.package/log/.package-2.log
================================================= log start ==================================================
running egg_info
writing vunit_hdl.egg-info/PKG-INFO
writing dependency_links to vunit_hdl.egg-info/dependency_links.txt
writing requirements to vunit_hdl.egg-info/requires.txt
writing top-level names to vunit_hdl.egg-info/top_level.txt
adding license file 'LICENSE.rst' (matched pattern 'LICEN[CS]E*')
error: [WinError 267] The directory name is invalid

================================================== log end =================================================== 
ERROR: FAIL could not package project - v = InvocationError("T:/vunit/vunit/.tox/.package/bin/python C:/msys64/mingw64/lib/python3.8/site-packages/tox/helper/build_requires.py setuptools.build_meta '' ''", 1)
```

@naveen521kk @Biswa96 any guess?
